cli/cmd/cluster.go

```diff
@@ -387,11 +509,10 @@ func newGetCredentialsCommand() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVar(&opts.namespace, "namespace", defaultMulticlusterNamespace, "the namespace in which the secret will be created")
-	cmd.Flags().StringVar(&opts.serviceAccountName, "service-account-name", defaultServiceAccountName, "the name of the service account")
-	cmd.Flags().StringVar(&opts.serviceAccountNamespace, "service-account-namespace", defaultMulticlusterNamespace, "the namespace in which the svc account resides on the remote cluster")
-	cmd.Flags().StringVar(&opts.clusterName, "cluster-name", defaultClusterName, "cluster name")
+	cmd.Flags().StringVar(&opts.namespace, "namespace", defaultMulticlusterNamespace, "The namespace for the service account")
+	cmd.Flags().StringVar(&opts.clusterName, "cluster-name", "", "cluster name")
```

Comment:
This starts to get confusing because there are several names in play now: 
* the service account name, which is also called the remote cluster name, which is represents the source cluster
* the cluster name, which represents the destination cluster, (the destination cluster is often referred to as the "remote cluster").

So a typical command might be 

```
linkerd --context west cluster link east --cluster-name west | kubectl --context east apply -f -
```

Is anyone going to get all those contexts and cluster names and service account names right?  It's very confusing which things refers to which thing.

At the very least we need to differentiate between service account names and cluster names.

```
linkerd --context west cluster allow east-sa | kubectl --context west apply -f -

linkerd --context west cluster link east-sa --cluster-name west | kubectl --context east apply -f -
```

This is a little better, but still pretty confusing.  

Better yet would be to give the cluster a fixed name at multicluster install time so that this doesn't get mixed up with the service account dance at all.

```
linkerd --context west cluster install west | kubectl --context west apply -f -

linkerd --context west cluster allow east | kubectl --context west apply -f -

linkerd --context west cluster link east | kubectl --context east apply -f -
```